### PR TITLE
add input to components package

### PR DESCRIPTION
**Pull request by SaraVieira**

### CODE REVIEW

File: packages/common/src/themes/codesandbox-black.js [hunk 1]
```diff
@@ -134,7 +134,7 @@ const colors = {
     background: tokens.grays[500],
     foreground: tokens.white,
     border: tokens.grays[800],
-    placeholderForeground: tokens.grays[300],
```
Comment (SaraVieira) [inline packages/common/src/themes/codesandbox-black.js]: I changed this because this is what matches the figma

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/3280/commits/b8bf3719b55c25e9320a25cf8803ef7f5cb2b63c is at https://pr3280.build.csb.dev/s/new.

File: packages/components/src/components/Input/index.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    color: 'input.foreground',
+    '::-webkit-input-placeholder': placeholderStyles,
+    '::-ms-input-placeholder': placeholderStyles,
+    '::placeholder': placeholderStyles,
+  })
+);
+
+const Label = styled.label(
+  css({
+    fontSize: 11,
```
Comment (siddharthkp) [inline packages/components/src/components/Input/index.tsx]: use the token danke

File: packages/components/src/components/Input/index.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+);
+
+interface IInputProps extends React.InputHTMLAttributes<HTMLInputElement> {
+  invisibleLabel?: string;
+  label?: string;
+}
+
+export const Input: React.FC<IInputProps> = ({
+  type = 'text',
+  invisibleLabel,
```
Comment (siddharthkp) [inline packages/components/src/components/Input/index.tsx]: should we automate this bit? If label isn't provided, use invisibleLabel?

File: packages/components/src/components/Input/index.tsx [hunk 4]
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import styled from 'styled-components';
+import css from '@styled-system/css';
+import { VisuallyHidden } from 'reakit/VisuallyHidden';
```
Comment (siddharthkp) [inline packages/components/src/components/Input/index.tsx]: If we have a choice can we use [reach/visually-hidden](https://reacttraining.com/reach-ui/visually-hidden)?

File: packages/components/src/components/Input/index.tsx [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+import { uniqueId } from 'lodash-es';
+
+const placeholderStyles = {
+  color: 'input.placeholderForeground',
+  fontSize: 13,
+};
+
+export const InputComponent = styled.input(
+  css({
```
Comment (siddharthkp) [inline packages/components/src/components/Input/index.tsx]: does input need a font size + font family explicitly?

File: packages/components/src/components/Input/index.tsx [hunk 6]
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import styled from 'styled-components';
+import css from '@styled-system/css';
+import { VisuallyHidden } from 'reakit/VisuallyHidden';
+import { uniqueId } from 'lodash-es';
+
+const placeholderStyles = {
+  color: 'input.placeholderForeground',
+  fontSize: 13,
```
Comment (siddharthkp) [inline packages/components/src/components/Input/index.tsx]:
```suggestion
  fontSize: 3,
```

Comment (SaraVieira) [reply packages/components/src/components/Input/index.tsx]: The token is the array index

OHHHHHHHHHHHHH

File: packages/components/src/components/Input/index.tsx (hunk 4 above)
Comment (SaraVieira) [reply packages/components/src/components/Input/index.tsx]: why though? ahah

File: packages/components/src/components/Input/index.tsx (hunk 5 above)
Comment (SaraVieira) [reply packages/components/src/components/Input/index.tsx]: Font family will be inter for all I thin, I will add the font-size

File: packages/components/src/components/Input/index.tsx (hunk 6 above)
Comment (siddharthkp) [reply packages/components/src/components/Input/index.tsx]: I should probably right the keys down 😅

File: packages/components/src/components/Input/index.tsx (hunk 5 above)
Comment (siddharthkp) [reply packages/components/src/components/Input/index.tsx]: Do we add this in the globals?

```
body, input, select, button: {
  font-family: Inter
}
```

File: packages/components/src/components/Input/index.tsx (hunk 4 above)
Comment (siddharthkp) [reply packages/components/src/components/Input/index.tsx]: personal preference, no strong reason

File: packages/components/src/components/Input/index.tsx (hunk 2 above)
Comment (SaraVieira) [reply packages/components/src/components/Input/index.tsx]: Done!

File: packages/components/src/components/Input/index.tsx (hunk 4 above)
Comment (SaraVieira) [reply packages/components/src/components/Input/index.tsx]: Changed :)

Comment (siddharthkp) [reply packages/components/src/components/Input/index.tsx]: Thanks <3

File: packages/common/src/design-language/theme.ts [hunk 7]
```diff
@@ -8,6 +8,7 @@ const theme = {
 
   // we use a 4 point grid
   space: [0, 4, 8, 12, 16, 20, 24, 28, 32, 36, 40],
+  sizes: [0, 4, 8, 12, 16, 20, 24, 28, 32, 36, 40],
```
Comment (siddharthkp) [inline packages/common/src/design-language/theme.ts]: heckyeah

**Pull request merged**